runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -345,6 +345,11 @@
 	 * The varbit that stores the players {@code AccountType}.
 	 */
 	ACCOUNT_TYPE(1777);
```

Comment:
Remove the semicolon here and replace with a comma, thats why you're failing the travisCI build